hasher-matcher-actioner/src/OpenMediaMatch/tests/test_api.py

```diff
@@ -153,6 +153,19 @@ def test_banks_add_hash(client: FlaskClient):
         },
     }
 
+def test_banks_delete_hash(client: FlaskClient):
+    bank_name = "NEW_BANK"
+    image_url = "https://github.com/facebook/ThreatExchange/blob/main/pdq/data/bridge-mods/aaa-orig.jpg?raw=true"
+
+    create_bank(client, bank_name)
+    add_hash_to_bank(client, bank_name, image_url, 1)
+
+    post_response = client.delete(
+        f"/c/bank/{bank_name}/content/1"
+    )
+
+    assert post_response.status_code == 200
```

Comment:
we should add one more check to make sure the content is actually gone (though I notice this is the case with other tests as well)!

Reply:
Yeah I had started to do this but after rebuilding the index in the test i'm getting a 503. It's also happening with test_banks_add_hash_index:

```
    def test_banks_add_hash_index(app: Flask, client: FlaskClient):
        bank_name = "NEW_BANK"
        bank_name_2 = "NEW_BANK_2"
        image_url = "https://github.com/facebook/ThreatExchange/blob/main/pdq/data/bridge-mods/aaa-orig.jpg?raw=true"
        image_url_2 = "https://github.com/facebook/ThreatExchange/blob/main/pdq/data/misc-images/c.png?raw=true"

        # Make two banks and add images to each bank
        create_bank(client, bank_name)
        add_hash_to_bank(client, bank_name, image_url, 1)
        create_bank(client, bank_name_2)
        add_hash_to_bank(client, bank_name, image_url_2, 2)

        storage = get_storage()
        # ensure index is empty to start with
        assert storage.get_signal_type_index(PdqSignal) is None

        # Build index
        build_all_indices(storage, storage, storage)

        # Test against first image
        post_response = client.get(
            f"/m/raw_lookup?signal_type=pdq&signal={IMAGE_URL_TO_PDQ[image_url]}"
        )
>       assert post_response.status_code == 200
E       assert 503 == 200
E        +  where 503 = <WrapperTestResponse streamed [503 SERVICE UNAVAILABLE]>.status_code
```

Let me know if you thinkn it's just a set up issue though

Reply:
I think this is a test issue - it might be related to your build_all_indices, though you can trace (via prints) why it's hitting the 503, but I suspect its because it doesn't think it has the newly built indices ready. 

You don't need to check whether or not a match will work - it's enough that the banking API thinks its gone.

Reply:
there's not a get content_id within curation.py so

1) is checking the # of deleted rows in the response sufficient
2) (assuming no, since we want to check if the content is still in the db) do you want me to add a separate GET method that we can use outside of requiring an index?

let me know if im missing something!